include/pci_bus.h

```diff
@@ -24,6 +24,16 @@
 
 #include "mem.h"
 
+// Start of PCI address space
+constexpr uint32_t PCI_MEMORY_BASE      = 0xc000'0000; // gives 3072 MB max RAM
```

Comment:
UpperCamelCase for public constants please.

Reply:
OK, changed.